harold/plugins/deploy.py

```diff
@@ -619,17 +673,17 @@ def status_all(self, irc, sender, channel):
                 irc.send_message(channel, "%s does not allow deploys :heavy_check_mark:" % salon.channel)
 
     @inlineCallbacks
-    def hold(self, irc, sender, channel, *reason):
+    def hold(self, irc, sender, channel, type=DeployHoldType.manually_triggered, *reason):
```

Comment:
adding a parameter here will mess up parsing from user input in chat (yeah, that's awful, sorry!). i'd just hard code `manually_triggered` in the call to `salon.hold()` rather than making it a param.